developer-docs/how-to-guide/How_to_upload_existing_content_using_sunbird.md

```diff
@@ -10,285 +10,190 @@ allowSearch: true
+| X-Authenticated-Userid | String         | Represents the registered User ID authorized to execute the API.                                                                                                                  | swan@gmail.com             | Yes          |
+| Authorization          | String         | To make use of the API, you require authorization. Raise a request to the administrator for the use of the API. You will receive the authorization key. Specify the key received. | a1234567H98085678zz234              | Yes          |
+| ts                     | String         | Time Stamp at which upload request was sent.                                                                                                                                      | 2018-09-05T10:55:22Z | Yes          |
+
+#### Path Parameters ####
+
+| Path Parameter | Parameter Type | Description                   | Sample Value            | Is Mandatory |
+|----------------|----------------|-------------------------------|-------------------------|--------------|
+| Content ID     | String         | Content ID of the Request URL. The content ID is generated by making a request using the [Create Content API](http://docs.sunbird.org/latest/apis/content/#operation/Create%20Content) | do_21258058227713638416 | Yes          |
+
+> Note: The path has the following format <base_url>/content/v1/upload/<content_id>. For example, using the sample value of the content ID parameter provided in the **Path Parameters** section and assuming that the Sunbird staging environment is used, the path will be: https://staging.open-sunbird.org/api/content/v1/upload/do_21258058227713638416 
+
+#### Form Data Parameters 
+
+> Note: Provide the following form data parameter, if you are uploading a file from your local machine.
+
+
+| Form Data Parameter | Parameter Type | Description                                          | Sample Value | Is Mandatory |
+|---------------------|----------------|------------------------------------------------------|--------------|--------------|
+| file                | String         | The name of the content file that you want to upload | sample.pdf   | Yes          |
+
+
+
+> Note: Provide the following form data parameter, if you are uploading content from YouTube or any web portal . 
+
+| Form Data Parameter | Parameter Type | Description                                          | Sample Value | Is Mandatory |
+|---------------------|----------------|------------------------------------------------------|--------------|--------------|
+| fileUrl                | String         | URL of the content to be uploaded | https://www.youtube.com/watch?v=jro17htKV7A   | Yes  
 
 
+ > Note: You can also set the parameter **artifactUrl** to point to the required content in the [Update Content API](apis/content/#operation/Update%20Content)
+
+#### Request Body (Upload Content API)
```

Comment:
Please create a full cURL request including the header and form-data parameters.

Reply:
@maheshkumargangula As per our last discussion, the information was updated. However, this is insufficient. Hence, request your help to create the full curl request including header and form-data parameters.

Reply:
@Sunitashelar @Basreena - Below is the cURL request with headers to make the content upload request.
```shell
curl -X POST \
  https://staging.open-sunbird.org/api/content/v1/upload/<CONTENT_ID> \
  -H 'Authorization: <AUTH_TOKEN>' \
  -H 'X-Authenticated-User-Token: <USER_TOKEN>' \
  -H 'cache-control: no-cache' \
  -H 'content-type: multipart/form-data;' \
  -F file=@/Users/john/Documents/test.pdf
```